packages/components/src/Components/DateFormat/helper.js

```diff
@@ -13,6 +14,7 @@ const relativeTimeTable = [
     { rightBound: month, description: date => `${Math.round(date / (day))} days ago` },
     { rightBound: 2 * day, description: () => '1 day ago' },
     { rightBound: day, description: date => `${Math.round(date / (hour))} hours ago` },
+    { rightBound: twoHours, description: () => '1 hour ago' },
     { rightBound: hour, description: date => `${Math.round(date / (minute))} minutes ago` },
```

Comment:
Can you change how this is generated to something more extensible? Right now only `1 hour` is correct, but what about `1 month ago`?
```JS
const formatTime = (time, msg) => `${time} ${msg}${time !== 1 ? 's' : ''} ago`;

{ rightBound: month, description: date => formatTime(Math.round(date / (day)), 'day') },
{ rightBound: day, description: date => formatTime(Math.round(date / (hour)), 'hour') },
{ rightBound: hour, description: date => formatTime(Math.round(date / (minute)), 'minute') },
```